### PR TITLE
fix install paths

### PR DESCRIPTION
**Pull request by tripzero**
CMAKE_INSTALL_LIBDIR already has prefix in it.  Without this change, intel.icd contains /usr/usr/lib64/ and should contain just /usr/lib64.

### CODE REVIEW

**Comment by tripzero** [comment]
@JacekDanecki FYI

**Comment by ArturHarasimiuk** [comment]
according to https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html:
```
CMAKE_INSTALL_<dir>      - destination for files of a given type
CMAKE_INSTALL_FULL_<dir> - corresponding absolute path
```
CMAKE_INSTALL_LIBDIR will contain relative path but intel.icd should contain absolute.

where do you observe issue with content of intel.icd? please provide more data: which Linux distribution, cmake version, cmake command line used.

**Comment by tripzero** [comment]
clearlinux.  Looks like I'm overriding CMAKE_LIBDIR_INSTALL on my cmake command line:

`cmake -G 'Unix Makefiles' -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_SBINDIR=/usr/bin -DCMAKE_INSTALL_LIBDIR=/usr/lib64 -DLIB_INSTALL_DIR=/usr/lib64 -DLIB_SUFFIX=64 -DCMAKE_AR=/usr/bin/gcc-ar -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_RANLIB=/usr/bin/gcc-ranlib .. -DBUILD_TYPE=Release -DSKIP_UNIT_TESTS=1`

I agree with the documentation so let me dig into why clearlinux is setting this default.

**Comment by tripzero** [comment]
I think using CMAKE_INSTALL_FULL_LIBDIR should work for either case.

File: package.cmake [hunk 1]
```diff
@@ -39,8 +39,8 @@ if(UNIX)
 
   set(OCL_ICD_RUNTIME_NAME ${CMAKE_SHARED_LIBRARY_PREFIX}${NEO_DLL_NAME_BASE}${CMAKE_SHARED_LIBRARY_SUFFIX})
   install(
-    CODE "file( WRITE ${IGDRCL_BINARY_DIR}/libintelopencl.conf \"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}\n\" )"
-    CODE "file( WRITE ${IGDRCL_BINARY_DIR}/intel.icd \"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/intel-opencl/${OCL_ICD_RUNTIME_NAME}\n\" )"
```
Comment (ArturHarasimiuk) [inline package.cmake]: please do not remove intel-opencl from path. shared library is installed into this folder, see line 36. If you need different behavior for ClearLinux, we already have detection and should set extra variable, like we do for _dir_etc in lines 27-32.

**Comment by tripzero** [comment]
Acknowledged. Was a merge mistake. I will fix

**Comment by ArturHarasimiuk** [comment]
merged as https://github.com/intel/compute-runtime/commit/a38aed4ad2490b5e82961b0d18b3bd134efefc88. Closiing

**Pull request closed without merging**